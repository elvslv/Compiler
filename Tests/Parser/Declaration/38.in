var a: integer;

const
  MaxN = 10;

type
  TT = integer;
  TLong = array[1..MaxN] of TT;
  TLongLong = array[1..20] of TLong;

var q: TLongLong;

type
  TMegaPoints = record x, y, z: TLongLong; a, b, c: real; end;

var w: TMegaPoints;