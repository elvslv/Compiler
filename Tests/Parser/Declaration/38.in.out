var A : INTEGER;
const MAXN = 10; 
type TT = INTEGER;
type TLONG = array [const 1..const MAXN] of TT;
type TLONGLONG = array [const 1..const 20] of TLONG;
var Q : TLONGLONG;
type TMEGAPOINTS = record 
    var A : REAL;
    var B : REAL;
    var C : REAL;
    var X : TLONGLONG;
    var Y : TLONGLONG;
    var Z : TLONGLONG;
end;
var W : TMEGAPOINTS;
